app/src/main/java/com/example/lhi06/residue/ReviewAdapter.java

```diff
@@ -42,12 +44,33 @@ public void onBindViewHolder(@NonNull final ReviewViewHolder holder, final int p
         holder.albumName.setText(review.getThisAlbum().getCollectionName());
         Picasso.with(holder.albumArt.getContext()).load(review.getThisAlbum().getArtworkUrl()).into(holder.albumArt);
 
-        holder.rootView.setOnClickListener(new View.OnClickListener() {
+        holder.albumArt.setOnClickListener(new View.OnClickListener() {
```

Comment:
im guessing that the rootView sets anything that hasnt been set to something different to whatever rootView is